trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require '/decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @decoration.correct_name.upcase[0...10]
```

Comment:
Again here, since we are already using the capitalize decorator, we don't need to use the `upcase` here. Kindly remove the `upcase` as it will return the whole name in uppercase.

Reply:
Implemented all the required changes